README_py.md

```diff
@@ -0,0 +1,26 @@
+# Welcome to a Toy PyBind11 Algorithm
+
+## Why?
+The goal of this project is to allow users to take an algorithm written in C++ and translate that into runnable Python code.
+This could allow a user to develop C++ code and run it on the EmotiBit's firmware, software, and through Python hitting three different metrics.
+
+## Installation Steps
```

Comment:
I think this doc in general requires a general restructuring.

For example, "steps to run example" requests user to clone the repo, but we already need it in the previous section to import the environment.

Reply:
I think reformatting it and creating sections that "resonate a flow" will improve readability.

Reply:
Resolving this comment since the doc has grown. Need to reevaluate.